spec/classes/foreman_proxy__plugin__dynflow_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe 'foreman_proxy::plugin::dynflow' do
+  describe 'with default settings' do
+    let :facts do
+      on_supported_os['redhat-7-x86_64']
```

Comment:
Per the discussion on other PR (https://github.com/theforeman/puppet-foreman/pull/351), I'll leave this as one OS unless you want it differently here